.github/workflows/publish-hotfix.yml

```diff
@@ -0,0 +1,153 @@
+# Description: this workflow publishes UI packages to `latest` tag on NPM.
+#
+# Triggered by: whenever a PR is merged to `hotfix` branch
+
+# If successful, we will publish all updated UI packages to NPM.
+
+name: Test and Publish / hotfix
+
+on:
+  push:
+    branches: [hotfix]
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: write # Used to push tags to GitHub
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      has-changesets: ${{ steps.has-changesets.outputs.has-changesets }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+      - name: Check if repo has unconsumed changesets
+        id: has-changesets
+        uses: ./.github/actions/has-changesets
+      # Note: if there are no changesets, that means that either
+      #         (1) A hot fix PR was just merged, or
+      #         (2) no commits with changesets have been merged after packages were last published.
```

Comment:
@wlee221 This seems dangerous if unrelated commits sneak into the branch. Can we make `hotfix` a protected branch and require approvals before a pr is able to merge in?

Reply:
For sure, let's make that a requirement. Maybe we can 2pr sometime before we merge this?

Reply:
The `hotfix` branch is disposable though remember, it's always created from the latest `Version Packages` PR and deleted at the end, I don't believe there are any dangers. I can remove the (2) comment if it's making it confusing

Reply:
Makes sense